src/components/metadata/styles/metadata.scss

```diff
@@ -247,3 +251,39 @@ $list-inline-spacing: 0.2em;
     display: none;
   }
 }
+
+.pipeline-metadata__plot {
+  height: 300px;
+  background-color: rgb(1, 1, 1);
```

Comment:
I think these colours should be replaced with their variables too. This one could probably just be solid black unless there's a need?